specs/core/0_beacon-chain.md

```diff
@@ -647,7 +647,7 @@ def get_previous_epoch(state: BeaconState) -> Epoch:
     Return the current epoch if it's genesis epoch.
     """
     current_epoch = get_current_epoch(state)
-    return (current_epoch - 1) if current_epoch > GENESIS_EPOCH else current_epoch
+    return GENESIS_EPOCH if current_epoch == GENESIS_EPOCH else current_epoch - 1
```

Comment:
Tiny tiny nitpick: the previous version looks fine to me, and slightly more efficient since we expect in most cases, `current_epoch` is greater than `GENESIS_EPOCH`.

Reply:
I was trying to make it clear that `current_epoch - 1` is not even "considered" unless `current_epoch != GENESIS`. Not too fussed either way :)

Reply:
So in python, the left side is never run if the conditional is False, but I think this makes it slightly clearer to be careful with the subtraction. I'm oky with as is